lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/service/KmsRsaDecrypter.java

```diff
@@ -74,6 +73,8 @@ public byte[] decrypt(
                     AlgorithmSupportMessage.unsupportedJWEAlgorithm(alg, supportedJWEAlgorithms()));
         }
 
+        var keyId = configService.getParameter(JAR_KMS_ENCRYPTION_KEY_ID);
```

Comment:
This means that the decrypter 'knows' about the config parameter for performing decryption (so it would be less reusable if we needed to decrypt other JWEs with a different key), but...
- We don't need to do this currently
- The previous method meant we needed to be aware of the different types of decrypter (the client needs to know about KMS key ids)

A 'better' solution might involve a factory being passed around (like with the signer), but it doesn't feel like a valuable refactoring right now.